packages/clerk-js/src/core/clerk.ts

```diff
@@ -136,6 +136,13 @@ const defaultOptions: ClerkOptions = {
   signUpForceRedirectUrl: undefined,
 };
 
+class ClerkMap extends Map {
+  name = 'clerk-map';
+
+  constructor() {
+    super();
+  }
+}
```

Comment:
What's the thought process behind extending `Map` here? In case we want to add unique functionality? If that's the cache, I would maybe call it `ClerkDataCache` or something. If we don't have plans to extend it, I'd maybe drop the `ClerkMap` class altogether.

Reply:
This will be replaced with a simple `Map`, I was using this for logging purposes in order to identify if the correct "cache" was being used